NtupleProducer/test/datasets_UL17.txt

```diff
@@ -0,0 +1,214 @@
+=== DATA_TAU_2017 ===
+/Tau/Run2017B-UL2017_MiniAODv2-v1/MINIAOD
+/Tau/Run2017C-UL2017_MiniAODv2-v1/MINIAOD
+/Tau/Run2017D-UL2017_MiniAODv2-v1/MINIAOD
+/Tau/Run2017E-UL2017_MiniAODv2-v1/MINIAOD
+/Tau/Run2017F-UL2017_MiniAODv2-v1/MINIAOD
+
+=== DATA_ELE_2017 ===
+/SingleElectron/Run2017B-UL2017_MiniAODv2-v1/MINIAOD
+/SingleElectron/Run2017C-UL2017_MiniAODv2-v1/MINIAOD
+/SingleElectron/Run2017D-UL2017_MiniAODv2-v1/MINIAOD
+/SingleElectron/Run2017E-UL2017_MiniAODv2-v1/MINIAOD
+/SingleElectron/Run2017F-UL2017_MiniAODv2-v1/MINIAOD
+
+=== DATA_MU_2017 ===
+/SingleMuon/Run2017B-UL2017_MiniAODv2-v1/MINIAOD
+/SingleMuon/Run2017C-UL2017_MiniAODv2-v1/MINIAOD
+/SingleMuon/Run2017D-UL2017_MiniAODv2-v1/MINIAOD
+/SingleMuon/Run2017E-UL2017_MiniAODv2-v1/MINIAOD
+/SingleMuon/Run2017F-UL2017_MiniAODv2-v1/MINIAOD
+
+=== DATA_MET_2017 ===
+/MET/Run2017B-UL2017_MiniAODv2-v1/MINIAOD
+/MET/Run2017C-UL2017_MiniAODv2-v1/MINIAOD
+/MET/Run2017D-UL2017_MiniAODv2-v1/MINIAOD
+/MET/Run2017E-UL2017_MiniAODv2-v1/MINIAOD
+/MET/Run2017F-UL2017_MiniAODv2-v1/MINIAOD
+
+=== BACKGROUNDS_TT_2017 ===
+/TTTo2L2Nu_TuneCP5_13TeV-powheg-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v1/MINIAODSIM
+/TTToSemiLeptonic_TuneCP5_13TeV-powheg-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v1/MINIAODSIM
+/TTToHadronic_TuneCP5_13TeV-powheg-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v2/MINIAODSIM
+
+=== BACKGROUNDS_WJETS_2017 ===
+/WJetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v1/MINIAODSIM
+/WJetsToLNu_HT-70To100_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v1/MINIAODSIM
+/WJetsToLNu_HT-100To200_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v1/MINIAODSIM
+/WJetsToLNu_HT-200To400_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v1/MINIAODSIM
+/WJetsToLNu_HT-400To600_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v1/MINIAODSIM
+/WJetsToLNu_HT-600To800_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v1/MINIAODSIM
+/WJetsToLNu_HT-800To1200_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v2/MINIAODSIM
+/WJetsToLNu_HT-1200To2500_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v1/MINIAODSIM
+/WJetsToLNu_HT-2500ToInf_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v2/MINIAODSIM
+
+=== BACKGROUNDS_DY_2017 ===
+/DYJetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v2/MINIAODSIM
+/DYJetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9_ext1-v1/MINIAODSIM
```

Comment:
I would not process two huge DY samples

Reply:
done in commit [ecb0cfe](https://github.com/LLRCMS/LLRHiggsTauTau/pull/134/commits/ecb0cfe415ae3cc22f57d06395aae5fa38ae8566), not sure why it's not marked as 'outdated' here